tests/testthat/test-arg.R

```diff
@@ -17,6 +17,14 @@ test_that("informative error message on partial match", {
   )
 })
 
+test_that("informative error message on a typo", {
+  myarg <- "continuuos"
+  expect_error(
```

Comment:
I _think_ it would be a little nice to use `verify_output()` here, and then provide quite a few tests.

Reply:
TIL about `verify_output()`